rma_recall/views/rma_recall_view.xml

```diff
@@ -0,0 +1,183 @@
+                              <group>
+                                <group>
+                                  <field name="product_id" />
+                                  <field name="qty" />
+                                  <field name="uom_id" groups="uom.group_uom" />
+                                  <field name="location_id" />
+                                  <field name="partner_id" />
+                                </group>
+                                <group>
+                                  <field name="rma_id" />
+                                  <field name="scrap_id" />
+                                </group>
+                              </group>
+                            </form>
+                        </field>
+                    </group>
+                </sheet>
+                <div class="oe_chatter">
+                    <field
+                        name="message_follower_ids"
+                        widget="mail_followers"
+                        groups="base.group_user"
+                    />
+                    <field name="message_ids" widget="mail_thread" />
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record id="rma_recall_tree" model="ir.ui.view">
+        <field name="name">rma.recall.tree</field>
+        <field name="model">rma.recall</field>
+        <field name="arch" type="xml">
+            <tree string="Recalls">
+                <field name="name" />
+                <field name="lot_id" />
+                <field name="product_id" />
+                <field name="rma_date" />
+                <field name="origin" />
+                <field name="state" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="rma_recall_search" model="ir.ui.view">
+        <field name="name">rma.recall.search</field>
+        <field name="model">rma.recall</field>
+        <field name="arch" type="xml">
+            <search string="Search Recalls">
+                <field
+                    name="name"
+                    filter_domain="[('name', 'ilike', self)]"
+                    string="Name"
+                />
+                <field name="lot_id" />
+                <field name="product_id" />
+                <field name="origin" />
+                <group name="groupby">
+                    <filter
+                        name="state"
+                        string="Status"
+                        context="{'group_by': 'state'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="rma_recall_action" model="ir.actions.act_window">
+        <field name="name">Recalls</field>
+        <field name="res_model">rma.recall</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <!-- Menu items -->
+    <menuitem
+        id="rma_recall_menu"
+        name="Recalls"
+        parent="rma.menu_rma_root"
+        sequence="20"
```

Comment:
Maybe change the sequence in order to have the Recalls menu after the customer and supplier RMA menus.